hat/settings.py

```diff
@@ -529,6 +529,7 @@ def sentry_error_sampler(_, hint):
         integrations=[DjangoIntegration()],
         send_default_pii=True,
         release=VERSION,
+        ignore_errors=["django.security.DisallowedHost"],
```

Comment:
I don't know if this will work?
Let's see.

Last time I did this, [I used `ignore_logger`](https://docs.sentry.io/platforms/python/integrations/logging/#ignoring-a-logger).

```python
from sentry_sdk.integrations.logging import ignore_logger

ignore_logger("django.security.DisallowedHost")
```

Reply:
Thanks! I think it worked for me on Trypelim, but let me double-check that

Reply:
@kemar I went to dig a little deeper and it works on the version we're using (https://github.com/getsentry/sentry-python/blob/1.38.0/sentry_sdk/client.py#L429). However it appears undocumented and so it'll probably cause issues next library upgrade, so I'll change it to using an `ignore_logger`.

Reply:
The `ignore_logger` was already added after a rebase, so I've just removed it.